server/util/log/gcp/gcp.go

```diff
@@ -0,0 +1,122 @@
+package gcp
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"strconv"
+	"time"
+
+	"cloud.google.com/go/logging"
+	"github.com/rs/zerolog"
+
+	logpb "google.golang.org/genproto/googleapis/logging/v2"
+	structpb "google.golang.org/protobuf/types/known/structpb"
+)
+
+var (
+	ProjectID = flag.String("app.log_gcp_project_id", "", "The project ID to log to in GCP (if any).")
+	LogID     = flag.String("app.log_gcp_log_id", "", "The log ID to log to in GCP (if any).")
```

Comment:
since these apply to the executor as well as the app, wdyt about putting this under a separate top-level `log:` namespace? so

```
log:
  gcp:
    project_id: ...
    log_id: ...
```

I also think it'd be nice (at some point) to alias `log.level` --> `app.log_level`

Reply:
ah I see we have a bunch of other `app.log_*` flags too, like `app.log_include_short_file_name`. Feel free to ignore :)

Reply:
I think adding aliases for these is a good idea, but outside the scope of this PR.